CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ Changes
 * [UI]: Updated sample details metadata so that long values will be broken up over multiple lines. (20.05.3);
 * [Developer]: Updated `babel-env` to fix a security vulnerability caused by dependency `elleptic`
 * [UI]: Updated UI for creating a sample within a project.
+* [Developer]: Updated yarn lock file.
```

Comment:
No need for the changelog entry for this.  It's not really a change, just re-fixing something that was already fixed.

Reply:
Removed in 36ad03d1e87153350184ab8948dd9100b61bb626